### PR TITLE
update Radarr to 5.0.3

### PR DESCRIPTION
**Pull request by ParthJadhav**
Tested on:
- [x] x86_64 -> Digital Ocean droplet
- [x] Arm64 -> Multipass Instance on Apple M2

### CODE REVIEW

File: radarr/umbrel-app.yml [hunk 1]
```diff
@@ -18,10 +18,14 @@ gallery:
   - 2.jpg
   - 3.jpg
 path: ""
+releaseNotes: >-
+  Not sure where to find the releaseNotes for the update,
+  releases page on Github seem to be including not much in it.
+
```
Comment (nmfretz) [inline radarr/umbrel-app.yml:24]: Ya, it looks like our only option is to mash something together from the changes shown in the Github releases. I would suggest that we just say something very simple about this update taking Radarr from 4.1.0.6175 all the way to v5.0.3.8127 and then mention any major changes that change the functionality of the app. For instance it looks like as of v5, UI authentication is mandatory. We can then link to the releases page.

**Review by nmfretz — COMMENTED** [review]
Thanks for this, Parth. 

I just tested and noticed that there is a similar issue to the [one encountered for Prowlarr](https://github.com/getumbrel/umbrel-apps/pull/725#issuecomment-1756314999), where the mac container does not set up radarr with transmission for new installs.

Also, I see that UI authentication is now mandatory. Have you checked that things still work correctly with this new requirement?

image

**Comment by ParthJadhav** [comment]
> Thanks for this, Parth.
> 
> I just tested and noticed that there is a similar issue to the [one encountered for Prowlarr](https://github.com/getumbrel/umbrel-apps/pull/725#issuecomment-1756314999), where the mac container does not set up radarr with transmission for new installs.
> 
> Also, I see that UI authentication is now mandatory. Have you checked that things still work correctly with this new requirement?
> 
> image

I tested it and it still works. Even with the login page; Transmission is configured properly.

File: radarr/umbrel-app.yml (hunk 1 above)
Comment (ParthJadhav) [reply radarr/umbrel-app.yml]: Okays, I updated the release notes !

**Comment by nmfretz** [comment]
Perfect, thanks Parth! Going live 🎬

**Pull request merged**